src/openrct2/scripting/ScNetwork.hpp

```diff
@@ -447,6 +449,38 @@ namespace OpenRCT2::Scripting
 #    endif
         }
 
+#    ifndef DISABLE_NETWORK
+        std::shared_ptr<ScSocketServer> createServer()
+        {
+            auto& scriptEngine = GetContext()->GetScriptEngine();
+            auto plugin = scriptEngine.GetExecInfo().GetCurrentPlugin();
+            auto socket = std::make_shared<ScSocketServer>(plugin);
+            scriptEngine.AddSocket(socket);
+            return socket;
+        }
+#    else
+        void createServer()
```

Comment:
Is it ok the two method have different signatures? Shouldn't this one return `std::shared_ptr<ScSocketServer> createServe{}`?

Reply:
This is conditional compilation though, so it isn't an overload.